### DISCUSSION TITLE
Feedback Page

### DISCUSSION THREAD

**Discussion by FetzerTony** [discord-tickets/bot#549] [Support (Q&A)]
Hey,
I wanted to ask why I can't see any feedback on the feedback page. Is this not done yet? And if so, when might this happen? I love the Bot so far. It is the best I could find for myself. Thanks for this great free bot.

Best regards
Tony

**Comment by eartharoid** [comment]
The portal isn't complete yet so the page doesn't exist. I don't know when it'll be done.
For now, you can access feedback individually with the [`/transcript`](https://discordtickets.app/features/commands/#transcript) command.
